### DISCUSSION TITLE
The parity of the length of the path of an /etc file...

### DISCUSSION THREAD

**Discussion by omertuc** [bootc-dev/bootc#952] [General]
... seems unimportant

Is this a bug / typo?

https://github.com/containers/bootc/blob/a3731457a22ffc0d412b3529a0a321a7031e7468/ostree-ext/src/fixture.rs#L247

**Comment by omertuc** [comment]
Clippy is complaining about this line. While trying to fix the clippy diagnostic, I tried to understand what the line even means, but it really seems non-sensical

**Comment by omertuc** [comment]
cc @cgwalters looks like your code originally

**Comment by cgwalters** [comment]
Clearly that code needed a comment, I don't remember for sure why I did that. But just staring at it my vague guess is that it was a way to vary the labels in a deterministic way, i.e. not everything in `/etc` is just `etc_t`.

So we can trivially change it to something that doesn't trigger a clippy lint.

BTW note this is just in test code.

**Comment by omertuc** [comment]
Ah fixture as in test fixture... OK, makes sense in that context

**Comment by omertuc** [comment]
https://github.com/containers/bootc/pull/954
